### PR TITLE
Add Install instructions, with a requirements.txt

### PR DESCRIPTION
**Pull request by gavinfernandes2012**
Merge this if you don't want me to write a setup.py file and turn this into a package.
I have another branch set up with a setup.py file, which allows installation using pip3. but currently,
the way this project is set up, the sdb-console and parsley grammars aren't included in the package.

If we were to make the package more like [this](https://github.com/BillMills/python-package-example) example package,
we could make this installable with pip. Then we could run the sdb-console using something like:
`python3 -m Semantic-DB`
and be dropped into a console from anywhere.

Tl;dr If you're fine with this, merge it, if you want to package the Semantic-DB, don't, and I'll move things around into a package.

### CODE REVIEW

**Comment by gavinfernandes2012** [comment]
I can confirm tkinter is a matplotlib dependency, and we can remove the dependency by switching to the agg matplotlib backend ([source](https://github.com/matplotlib/matplotlib/issues/7115/)). D'you want me to move forward with this change?

And can't we just ```pip3 install graphviz``` ([source](https://pypi.org/project/graphviz/))

I'll go ahead and work on packaging it, I'm hoping sublime's multiple cursors will come to the rescue here.

**Comment by gavinfernandes2012** [comment]
Ohh and I just wanted to confirm, comprehensive-tests.py is really still comprehensive right?
I've been using it to test whether the project works, and couldn't we split the tests into unit tests?

**Comment by gavinfernandes2012** [comment]
PS: I'll open another pull request from the relevant branch on my fork, closing this request now.

**Comment by gavinfernandes2012** [comment]
Reply to the above comments here: #3
